src/types.ts

```diff
@@ -47,6 +47,10 @@ export interface DesignData {
   structureKeyValueData: KeyValueData
 }
 
+/** An array of unique UUIDv4 identifier for the selected designs. */
+export interface SelectedDesignIds extends Array<string> {
+}
```

Comment:
**issue:** interface is defining a derivative of `Array` not a specific `Array` format

```suggestion
export type SelectedDesignIds = Array<string>
```